src/main/java/frc/robot/subsystems/instances/TimeOfFlight.java

```diff
@@ -0,0 +1,23 @@
+package main.java.frc.robot.subsystems.instances;
+
+public class TimeOfFlight extends SybsystemBase implements TimeOfFlightInterface{
```

Comment:
What and why is this?

Reply:
Time of flight subsystem and interface has been deleted